drivers/SmartThings/matter-switch/src/eve-energy/init.lua

```diff
@@ -186,9 +251,14 @@ end
 -------------------------------------------------------------------------------------
 
 local function device_init(driver, device)
-  log.info_with({ hub_logs = true }, "device init")
+  if not device:get_field(COMPONENT_TO_ENDPOINT_MAP) and
```

Comment:
```suggestion
if device.network_type == device_lib.NETWORK_TYPE_MATTER then
  if not device:get_field(COMPONENT_TO_ENDPOINT_MAP) and
```

We want this check so that we only run this function for the parent device. You can have this check around the entirety of the init function.

Reply:
Change implemented, thanks for the review!